src/elements/content-picker/Footer.scss

```diff
@@ -9,11 +9,23 @@
     justify-content: space-between;
     padding: 0 20px;
 
+    .bcp-footer-left {
+        display: flex;
+        flex: 1;
+        min-width: 0;
```

Comment:
Is this necessary? Is it overriding something?

Reply:
The button won't resize without it: following advice from stack overflow / https://css-tricks.com/flexbox-truncated-text/